clusterloader2/pkg/measurement/manager.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package measurement
+
+import (
+	"sync"
+
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// MeasurementManager manages all measurements calls and executions.
+type MeasurementManager struct {
+	clientSet    clientset.Interface
+	lock         sync.Mutex
+	// measurements is a map: methodName -> identifier -> Measurement.
+	// First level divides measurements into method classes, second into instances.
```

Comment:
I think this comment is a bit redundant. Maybe simply say 'Map from method type and identifier to measurement instance'?

Reply:
done